### DISCUSSION TITLE
Opção de visualizar quem fez o ultima alteração de OS.

### DISCUSSION THREAD

**Discussion by almeida-pf** [RamonSilva20/mapos#1773] [Ideas]
Poderia ter opção de visualizar qual usuário alterou a OS, como se fosse um Histórico mesmo porém disponível para todos pois assim não corre o risco do usuário alterar algo sem que fique um registro.

**Comment by willph** [comment]
> Poderia ter opção de visualizar qual usuário alterou a OS, como se fosse um Histórico mesmo porém disponível para todos pois assim não corre o risco do usuário alterar algo sem que fique um registro.

Tem log de auditoria no menu configurações.

**Reply by almeida-pf** [reply]
Seria interessante se pudesse ficar visível, algo parecido como a do sistema que uso, pois como temos muitos usuários as vezes fica chato vc ter que ficar pedindo para o ADM ver quem fez tal coisa ou quem alterou um status por ultimo. 

![image](https://user-images.githubusercontent.com/31329742/164093222-71c4af9a-1f14-4dfc-80f2-a1cf26687fa3.png)

**Reply by borgesthiago** [reply]
Concordo! Ficaria muito mais rápido a visualização e de maneira colaborativa.

**Comment by Pr3d4dor** [comment]
@almeida-pf @borgesthiago Basta você adicionar essa permissão para os demais usuários.
